index.js

```diff
@@ -1,16 +1,36 @@
-/***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
+header.style.color = "red"
 
 
 /***** Deliverable 3 *****/
-// console.log("the player object looks like this:", player)
+console.log("the player object looks like this:", player.name)
+document.querySelector("h2").textContent = player.name
+document.querySelector("img").src = player.photo
+document.querySelector("em").textContent = player.nickname
+document.querySelector("img").alt = player.photo
```

Comment:
V concise! Just pointing out that you're accessing the same `img` element twice with querySelector, which means the browser has to find that element twice - might be worth saving it to a variable to avoid an unnecessary lookup (but that is an extremely minor optimization).